runtime/model-protobuf/src/main/java/io/aklivity/zilla/runtime/model/protobuf/config/ProtobufModelConfigBuilder.java

```diff
@@ -29,6 +29,7 @@ public class ProtobufModelConfigBuilder<T> extends ConfigBuilder<T, ProtobufMode
     private List<CatalogedConfig> catalogs;
     private String subject;
     private String view;
+    private String syntax;
```

Comment:
The protobuf model is independent of syntax, since we support both `proto2` and `proto3` after these changes, so no need to track syntax here, agree?